backend/src/main/java/storage/repository/GenericRepository.java

```diff
@@ -5,6 +5,7 @@
 import com.google.gson.JsonObject;
 import com.google.gson.JsonPrimitive;
 import model.DataModel;
+import org.postgresql.util.PSQLException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.postgresql.util.PSQLException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)